### DISCUSSION TITLE
Sync Error with 2023.12.0 Desktop and Browser Extension

### DISCUSSION THREAD

**Discussion by m4rquez** [dani-garcia/vaultwarden#4178] [Q&A] [answered]
After Updating my Desktop App and Chrome Browser Extension to **2023.12.0**, both are no longer syncing successful. 
Vaultwarden Server Version is **1.30.1**

There are no Errors in the Server Log, just the expected "200", same in the Desktop App Dev Tools
![image](https://github.com/dani-garcia/vaultwarden/assets/20598399/556d26e8-75dd-4b83-8afa-2523a6133dcd)

After downgrading to **2023.10.1 Sync is working** again.
![image](https://github.com/dani-garcia/vaultwarden/assets/20598399/bda027bc-202e-4fd4-bfcb-ccb919c1135b)

Maybe there is a breaking change in the official Bitwarden implementation?

**Comment by BlackDex** [comment, marked as answer]
See #4152  and #4174

**Reply by m4rquez** [reply]
how could i've missed that. SORRY 🤦‍♂️
